homeassistant/components/cloud/client.py

```diff
@@ -106,20 +107,35 @@ async def logged_in(self) -> None:
         """When user logs in."""
         await self.prefs.async_set_username(self.cloud.username)
 
-        if self.alexa_config.enabled and self.alexa_config.should_report_state:
+        async def enable_alexa(_):
+            """Enable Alexa."""
             try:
                 await self.alexa_config.async_enable_proactive_mode()
+            except aiohttp.ClientError:  # If no internet available yet
+                async_call_later(self._hass, 30, enable_alexa)
```

Comment:
This will run forever if we always get a ClientError, we should at least log a warning and/or limit this

Reply:
I will now print a warning if it's retrying and HA is not running yet. That way if internet is still being established on the machine, it won't log.

Reply:
Perfect :+1: